core/server/adapters/scheduling/SchedulingDefault.js

```diff
@@ -197,61 +197,61 @@ SchedulingDefault.prototype._execute = function (jobs) {
  * - if we detect to publish a post in the past (case blog is down), we add a force flag
  */
 SchedulingDefault.prototype._pingUrl = function (object) {
-    debug('Ping url', object.url, moment().format('YYYY-MM-DD HH:mm:ss'), moment(object.time).format('YYYY-MM-DD HH:mm:ss'));
+    const {url, time} = object;
+
+    debug('Ping url', url, moment().format('YYYY-MM-DD HH:mm:ss'), moment(time).format('YYYY-MM-DD HH:mm:ss'));
 
     let timeout;
```

Comment:
@vikaspotluri123 this variable is not used anymore after removal of clearTimeout :wink:

Reply:
Oops 🙈 🙄